crates/ibc/src/core/ics04_channel/commitment.rs

```diff
@@ -86,31 +86,51 @@ pub(crate) fn compute_packet_commitment(
     timeout_height: &TimeoutHeight,
     timeout_timestamp: &Timestamp,
 ) -> PacketCommitment {
-    let mut hash_input = timeout_timestamp.nanoseconds().to_be_bytes().to_vec();
+    let mut hash_input = [0; 3 * 8 + 32];
```

Comment:
```suggestion
    let mut hash_input = [0; 8 * 3 + 32];
```

Just being pedantic as it is 8 bytes x3 and 32 bytes x1.